pippo-session-parent/pippo-session-hazelcast/src/test/java/ro/pippo/session/hazelcast/HazelcastSessionDataStorageTest.java

```diff
@@ -65,18 +65,19 @@ public void testCreate() {
      */
     @Test
     public void testSave() {
-        HazelcastInstance hazelcastInstance2 = Hazelcast.newHazelcastInstance();
+//        HazelcastInstance hazelcastInstance2 = Hazelcast.newHazelcastInstance();
```

Comment:
@decebals I think this may be due this Hazelcast [issue](https://github.com/hazelcast/hazelcast/issues/8871)